person.rb

```diff
@@ -7,12 +9,17 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @correct_name = Corrector.new
   end
 
   def can_use_services?
     return false unless is_of_age? || @parent_permission
   end
 
+  def validate_name(name)
+    @correct_name.correct_name(name)
+  end
```

Comment:
- [ ] the value this returns should be assigned to the `@name` instance variable, after that it will look something like below.
```
def validate_name(name)
    @name = @correct_name.correct_name(@name)
end
```